test/app.test.js

```diff
@@ -14,8 +14,14 @@ var common = require('./common');
 var assert = require('assert');
 var expect = require('chai').expect;
 var fs = require('fs');
+var rimraf = require('rimraf');
+
+// Rename it to run as the last test
```

Comment:
nitpick: can we re-phrase this statement to start like this: `The test file is named app.test.js as a workaround for...`?